### ISSUE TITLE
Tabs not functioning in Legacy browsers: Firefox , Safari and IE

### ISSUE THREAD

**Issue by eoghanTadhg** [enove/NoSLB#173]
It seems the `check` function is causing some problems, @micchickenburger .
See this article: https://github.com/meteor/meteor/issues/3207

This is observable in `tabs.js` and may be applicable elsewhere.
`check(event, $.Event);` is returning:

`Error: Match error: Expected plain object`

**Comment by micchickenburger** [comment]
Not reproducible on Safari 10.0.1 or Firefox 49.0.1 on Mac OS X 10.12.1.

**Comment by eoghanTadhg** [comment]
I suppose you don't have access to Safari Version 9.1 (11601.5.17.1)

**Comment by eoghanTadhg** [comment]
![image](https://cloud.githubusercontent.com/assets/5591794/20329923/1ef45590-ab50-11e6-93f6-8e8e966574dc.png)

**Comment by eoghanTadhg** [comment]
@micchickenburger

**Comment by eoghanTadhg** [comment]
Same error message as ie11. Screnshot above provided from Safari.

**Comment by micchickenburger** [comment]
Yes, I was able to reproduce the issue in IE 11 on Windows 10.  The problem is with the check library.  It does a check to see whether the pattern is a function.  Since constructors are functions, this is fine for pretty much any instantiated object.  However, older browsers treat built-in components like `Element` as non-function objects, despite there being instantiations of them.

In older-recent versions of Firefox, Safari, and IE these browsers consider `Element instanceof Function // false`.  Conversely, in new versions of those browsers and in Chrome `Element instanceof Function // true`.  That allows checks to pass for elements only in new browsers:

```
check(document.createElement('a'), Element);
// new browsers, no problem
// old browsers:  Error:  Match Error:  Expected plain object
```

**Comment by eoghanTadhg** [comment]
This issue is occurring on Firefox latest now I've observed.
However I do not think this is related to the tabs displaying.

**Comment by eoghanTadhg** [comment]
Retract the last statement. The check function is preventing the toggling of `aria-expanded` and `aria-hidden` in ie11.

**Comment by eoghanTadhg** [comment]
So regarding your last comment, @micchickenburger : How should we proceed?
Were you intending on writing a fallback script or to just not support IE11 and other legacy browsers?

**Comment by eoghanTadhg** [comment]
Updates for this issue are handled on the branch `qa-legacy-browsers`.
I believe I've done all I can for this one. The last remaining issue pertains to the check script.
